Aruba Network/aruba-os/ingest/parser.yml

```diff
@@ -4,7 +4,32 @@ pipeline:
     external:
       name: grok.match
       properties:
-        pattern: '%{DATA:prefix}\:\s*%{USERNAME:obs}\s*\:\s*%{GREEDYDATA:payload}\s*'
+        pattern: "%{SYSLOG}|%{DEFAULT}"
+        custom_patterns:
+          SYSLOG: '<%{NUMBER:code}> <%{NUMBER}> (<%{WORD}> )?<%{WORD} %{IP:src_ip}>\s*(%{WORD}:\s*)?%{GREEDYDATA:syslog_payload}'
+          DEFAULT: '%{DATA:prefix}\:\s*%{USERNAME:obs}\s*\:\s*%{GREEDYDATA:payload}\s*'
+
+  - name: parse_syslog_message
+    external:
+      name: grok.match
+      properties:
+        input_field: "parse_message.message.syslog_payload"
+        pattern: "%{FORMAT_133121}|%{KV_PART}|%{DEFAULT}"
+        custom_patterns:
+          FORMAT_133121: "%{DATA} to %{IP:dst_ip}(:%{NUMBER:dst_port})? with %{GREEDYDATA}"
+          KV_PART: '%{DATA:payload}.\s*%{GREEDYDATA:kv_part}'
+          DEFAULT: "User %{USERNAME:user_name}.*?"
```

Comment:
Sadly, according to the test, the username is not extracted :(.
Can you take a look at it? Thank you.